### DISCUSSION TITLE
How do I map Expand/Shrink Selection (normally <A-up> and <A-down>)

### DISCUSSION THREAD

**Discussion by agingorange** [JetBrains/ideavim#826] [Q&A] [answered]
In my Neovim configuration, I use v and V resp. to expand and shrink the selection using Treesitter. So, to kind of get the same, I tried the following:

`vmap v <Action>GuiDesigner.ExpandSelection`

`vmap v <A-up>`

Neither worked. Is it possible to do the remap I'm looking for?

**Comment by lippfi** [comment]
Action should be wrapped in braces
`vmap v <Action>(GuiDesigner.ExpandSelection)`
Does it work?

**Reply by agingorange** [reply]
It didn't work. I tried it in IntelliJ Ultimate/EAP and the RustRover Preview. Alt Up/Down works in either of them.

**Reply by lippfi** [reply, marked as answer]
Ok, I get it now, it's a wrong action. Please see https://github.com/JetBrains/ideavim?tab=readme-ov-file#finding-action-ids
I've just tracked id of expanding selection and it is `EditorSelectWord` 
`vmap .. <Action>(EditorSelectWord)` should work

**Reply by agingorange** [reply]
Excellent, thanks! I checked with :actionlist but obviously that was something else.
